python file>.py

```diff
@@ -0,0 +1,57 @@
+fs_pairs = {'bob': 'bobby',
+            'apple': 'tree',
+            'dog': 'wolf'}
+choice = None
+
+while choice != '1':
+    print('''
+
+    Your choices right now:
+
+    1: Exit
+    2: add son father pair
+    3: replace a son's father with another 
+    4: delete son father pair
+    5: view son father pairs
```

Comment:
nit, but keep sentence casing consistent:
```suggestion
    1: Exit
    2: Add son father pair
    3: Replace a son's father with another 
    4: Delete son father pair
    5: View son father pairs
```